analysis/report_templates/default.html

```diff
@@ -232,17 +232,51 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                 </li>
             </ul>
 
-            {% if coverage_report %}
+            {% if detailed_coverage_report %}
+            <ul class="collapsible">
+                <li>
+                    <div class="collapsible-header">
+                        Differential matrix
+                    </div>
+                    <div class="collapsible-body">
+
+                        <div class="row">
+                            <div class="col s6 offset-s3">
+                                <h5 class="center-align">Ranking by unique edges covered</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.unique_region_ranking_plot }}">
+                                    The table shows the unique edges coverage rank of each fuzzer
+                                    on this benchmark. Unique edges are edges only covered by one
+                                    certain fuzzer.
+                            </div>
+                        </div> <!-- row -->
+
+                        <div class="row">
+                            <div class="col s6 offset-s3">
+                                <h5 class="center-align">Complementary pairs table</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.complementary_pairs_plot }}">
+                                    This table is a square matrix where each row and column
+                                    represents a fuzzer, and each cell contains a number showing the
+                                    number of edges covered by the fuzzer of the column but not by
+                                    the fuzzer of the row.
+                            </div>
+                        </div> <!-- row -->
+
+                    </div>
+                </li>
+            </ul>
+
             <ul class="collapsible">
                 <li>
                     <div class="collapsible-header">
                         Coverage reports for each fuzzer on this benchmark
                     </div>
                     <div class="collapsible-body">
                         <div class="row">
-                            {% for fuzzer in benchmark.fuzzers %}
+                            {% for fuzzer in benchmark._fuzzer_names %}
                             <div class="col">
-                                <a class="waves-effect waves-light btn" href="https://storage.googleapis.com/{{ experiment_filestore_name }}/{{ experiment.name }}/reports/coverage/{{ benchmark.name }}/{{ fuzzer }}/index.html">{{ fuzzer }}</a>
+                                <a class="waves-effect waves-light btn" href="https://storage.googleapis.com/{{ benchmark._get_filestore_name(fuzzer) }}/coverage/reports/{{ benchmark.name }}/{{ fuzzer }}/index.html">{{ fuzzer }}</a>
```

Comment:
if we are calling _get_filestore_name externally, please remove the `_` prefix marking it internal